src/iotjs_env.c

```diff
@@ -152,6 +153,15 @@ uv_loop_t* iotjs_environment_loop(const iotjs_environment_t* env) {
   return _this->loop;
 }
 
+void iotjs_environment_set_dbgout(iotjs_environment_t* env, dbgout_t dbgout) {
+  IOTJS_VALIDATED_STRUCT_METHOD(iotjs_environment_t, env);
```

Comment:
@daeyeon Currently we are removing `IOTJS_VALIDATED_STRUCT_XXX`. We can remove it.

Reply:
@daeyeon I don't want to mix 'validated struct' removal patch with your PR. So I pushed #1432.

Reply:
I agree. It should be handled in another pr.